### ISSUE TITLE
Improve the accessibility rating

### ISSUE THREAD

**Issue by mateuszmigas** [mateuszmigas/painting-droid#28]
**Is your feature request related to a problem? Please describe.**
The app is not achieving a high score in terms of accessibility; there are some missing elements.

![CleanShot 2024-04-24 at 20 30 23@2x](https://github.com/mateuszmigas/painting-droid/assets/54471371/bfb62a56-0e8a-47c9-86f1-18f9dda31822)

**Describe the solution you'd like**
Add missing attributes to improve accessibility.

**Describe alternatives you've considered**
N/Ak

**Additional context**
N/A
